<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Application Paths</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QGroupBox" name="appVer_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Applications and versions</string>
     </property>
     <layout class="QFormLayout" name="appVer_formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="appName_label">
        <property name="text">
         <string>Application:</string>
        </property>
        <property name="buddy">
         <cstring>appName_comboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="appVer_label">
        <property name="text">
         <string>Version:</string>
        </property>
        <property name="buddy">
         <cstring>appVer_comboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="appVer_frame">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="appVer_horizontalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="appVer_comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Select a version for this application, or add a new version by typing a new name and pressing Enter</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="mandatoryField" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="appVerDel_toolButton">
           <property name="toolTip">
            <string>Remove this version of the application</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="appName_frame">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="appName_horizontalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="appName_comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Select an application, or add a new application by typing a new name and pressing Enter</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="mandatoryField" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="appNameDel_toolButton">
           <property name="toolTip">
            <string>Remove this application</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="execPaths_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Executable paths</string>
     </property>
     <layout class="QFormLayout" name="execPaths_formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="winPath_label">
        <property name="text">
         <string>Windows:</string>
        </property>
        <property name="buddy">
         <cstring>winPath_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="osxPath_label">
        <property name="text">
         <string>Mac:</string>
        </property>
        <property name="buddy">
         <cstring>osxPath_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="linuxPath_label">
        <property name="text">
         <string>Linux:</string>
        </property>
        <property name="buddy">
         <cstring>linuxPath_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="guess_pushButton">
        <property name="toolTip">
         <string>Try to guess the executable paths based on the template</string>
        </property>
        <property name="text">
         <string>Guess</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="osxPath_frame">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="osxPath_horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="osxPath_lineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="osxPathBrowse_toolButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="noBackground" stdset="0">
            <bool>false</bool>
           </property>
           <property name="attach" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="winPath_frame">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="winPath_horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="winPath_lineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="winPathBrowse_toolButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="noBackground" stdset="0">
            <bool>false</bool>
           </property>
           <property name="attach" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="linuxPath_frame">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="linuxPath_horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="linuxPath_lineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="linuxPathBrowse_toolButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="noBackground" stdset="0">
            <bool>false</bool>
           </property>
           <property name="attach" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="appPaths_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>appPaths_buttonBox</tabstop>
  <tabstop>appName_comboBox</tabstop>
  <tabstop>appNameDel_toolButton</tabstop>
  <tabstop>appVer_comboBox</tabstop>
  <tabstop>appVerDel_toolButton</tabstop>
  <tabstop>winPath_lineEdit</tabstop>
  <tabstop>winPathBrowse_toolButton</tabstop>
  <tabstop>osxPath_lineEdit</tabstop>
  <tabstop>osxPathBrowse_toolButton</tabstop>
  <tabstop>linuxPath_lineEdit</tabstop>
  <tabstop>linuxPathBrowse_toolButton</tabstop>
  <tabstop>guess_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
